AEPMessaging/Sources/Message.swift

```diff
@@ -53,11 +54,11 @@ public class Message {
     ///   - event: the Rules Consequence `Event` that defines the message and contains reporting information
     init(parent: Messaging, event: Event) {
         self.parent = parent
-        triggeringEvent = event
```

Comment:
line 45: property definition
Should the triggeringEvent has to be optional (Event?) ?
Looks like we always initialize with an non optional `event`

Reply:
good point - it will never be `nil` so i've updated the definition for `triggeringEvent` to reflect this.